airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.73.1
```

Comment:
no need to update the changelog. this is done automatically by the publish github action

Reply:
Fixed